src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs

```diff
@@ -55,6 +55,9 @@ internal sealed class ServiceFunction : BuiltinFunction, IAsyncTexlFunction2, IH
         // Check will return a warning when used
         private readonly bool _isDeprecated;
 
+        // For functions that have "x-ms-visibility" set to "internal" in swagger file.
+        private readonly bool _isInternal;
```

Comment:
How is PowerApps handling this?

They must hit the same issue when importing Sharepoint swagger.

Reply:
They do